pkgs/by-name/ni/nist-feed/package.nix

```
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, bash
+, jq
+, killall
+, libnotify
+, cron
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nist-feed";
+  version = "unstable-2024-01-20";
```

There's some new versioning guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

```suggestion
  version = "0-unstable-2024-01-20";
```